tedge/src/cli/connect/command.rs

```diff
@@ -421,6 +427,10 @@ fn new_bridge(
         return Err(err);
     }
 
+    if let Err(err) = service_manager_result {
+        return Err(err.into());
```

Comment:
This error should be addressed as a special case where `new_bridge()` is called. As of now, if systemctl doesn't exist, the message that user gets is below.

```
$ sudo ./tedge connect c8y
Checking if systemd is available.

It appears that 'systemd' service manager is not installed on the system, thin-edge.io works seamlessly with 'systemd'.
'tedge connect' will configure the necessary tedge components, but you will have to start the required services on your own.
Start/restart mosquitto and other thin edge components.
Checking if configuration for requested bridge already exists.

Validating the bridge certificates.

Saving configuration for requested bridge.

Error: failed to create bridge to connect Cumulocity cloud.

Caused by:
    Service Manager: 'systemd' is not available on the system or elevated permissions have not been granted.

```

`Error: failed to create bridge to connect Cumulocity cloud.` isn't nice. We actually create a bridge.

Reply:
Disagree, as we don't create the vridge, but only configuration to enable the bridge, the user has to create the bridge.
I'm ok to change error message to indicate that some steps have failed.

Reply:
Looking at the whole error log, I think it can indeed be misleading.

I propose to:
1) Move the long explanation to end: "since systemd is not available, you will have to ...."
2) After the check for system; just say "Warning, systemd not available"
3) Change the error message to "Failed to connect Cumulocity cloud"

Reply:
Now it looks good, thanks.